route/src/route_generator_worker.cpp

```diff
@@ -69,7 +69,24 @@ namespace route {
                         auto full_file_name = itr->path().filename().generic_string();
                         cav_msgs::Route route_msg;
                         // assume route files ending with ".csv", before that is the actual route name
-                        route_msg.route_name = full_file_name.substr(0, full_file_name.find(".csv"));
+                        route_msg.route_id = full_file_name.substr(0, full_file_name.find(".csv"));
```

Comment:
Up on Line 58 there's still mention of ROUTE_SELECTION instead of SELECTION. Please make sure to update all the old state and event names (find all in the solution) to make sure it's been updated. Not sure if Guidance has any that needs to be updateda s well.